plugins/embed-optimizer/readme.txt

```diff
@@ -0,0 +1,52 @@
+=== Embed Optimizer ===
+
+Contributors:      wordpressdotorg
+Requires at least: 6.3
+Tested up to:      6.5
+Requires PHP:      7.0
+Stable tag:        0.1.0
+License:           GPLv2 or later
+License URI:       https://www.gnu.org/licenses/gpl-2.0.html
+Tags:              performance, images
```

Comment:
Tags should be changed for this plugin `embeds`?

Reply:
Done in 6b8d72a